.cascade-code/left-nav-refactor/_components/left_nav/left_nav_no_siblings_format

```diff
@@ -0,0 +1,73 @@
+## Format used by index pages where siblings don't want to be shown.
```

Comment:
This format somewhat mimics what `left_nav_format` does, except one is for index pages and one is for non-index pages.

With the index pages, the velocity format renders only children. With non-index pages, the velocity format renders only siblings.

We currently have a separate velocity format for not showing siblings (for index pages). It works fine with the new block, so I just refactored the existing one quite a bit. On the other hand, the primary left nav velocity format does NOT work with the new block. But it will work for pages that aren't index pages. So that's how I justified making a new separate format for non-index pages to use, in addition to the new block.

Reply:
@kaylaziegler Hopefully I'm wrong about what I think you're proposing...  but if you're hoping to have 2 blocks and 2 formats and clients set one of the other each time they create a new page, or that we go through and change all the existing pages to use one or the other depending on if the page is index or non-index, then I don't think that's going to be a reasonable solution.  The LeftNavNoSibling is only on a select few pages and was done by the Web Team, not clients.  Don't model the new solution after that concept. A Config Set for a template like the 2 Column Modular can only have 1 block and 1 format for the LEFT NAV region and should apply to all pages no matter what page name is given to resulting page. Maybe we can talk more tomorrow

Reply:
Yes, this had crossed my mind, too. There's still value in what's presented here: it cleans up the code and makes future changes easier. 

Still, based on what @mandythomas points out, it sounds like we'll have to explore other options to find a viable general solution to the slow-loading problem.

Reply:
We could potentially combine the `left_nav_no_siblings_format` and the `left_nav_format` and then just have two.

I didn't know we were not okay with having more than one format the way we do now. That was never brought up. The LeftNavNoSiblings very much needed to be rewritten anyways, if we are still planning to keep it.

I can see how having more than one block might be potentially difficult to implement. It's just incredibly inefficient now and kind of ridiculous that we're using such an unnecessarily large block for all the pages that aren't indexes with siblings.

My recommendation then would be to sacrifice the functionality of those index pages that show siblings and not have the dropdowns for the siblings children, and have every left nav on every page use the smaller block I've proposed. Non-index pages and pages that use the no-siblings format would still behave the same.

Reply:
On this point, I agree:

> I didn't know we were not okay with having more than one format the way we do now. 

But then discovering these sorts of things of one of the key parts of these exercises. We should try to document this somewhere where it won't be lost to future generations. :)

Which reminds me of one other suggestion I mean to make: a README file. Actually, perhaps two:

- A `README` file under `.cascade-code` that explains the purpose of this directory.
- A `README` file under `left-nav-refactor` that explain goals/motivation in general terms for this code.